lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1038,21 +1038,38 @@ SPIRVValue *SPIRVModuleImpl::addConstant(SPIRVType *Ty, uint64_t V) {
   return addConstant(new SPIRVConstant(this, Ty, getId(), V));
 }
 
+template <spv::Op OC>
```

Comment:
Do we really need this method definition in this file? It seems more logical to have it in SPIRVValue.[h|cpp]
And the same question for the constructor below.

Reply:
Moved to SPIRVValue.cpp